Controllers/UserControllers/insertAddress.js

```diff
@@ -0,0 +1,56 @@
+import sequelize from '../../db';
+
+const insertAddress = async (req, res) => {
+  try {
+    const { userId } = req.body;
+    if (userId) {
```

Comment:
what are you doing if `!userid`? 

I prefer to do something like
```
if (!userId) {
  // handle error case
  return;
}
```

rather than have the majority of the method in an `if` block